src/SmartComponents/Compliance/ComplianceCard.js

```diff
@@ -60,15 +61,18 @@ class ComplianceCard extends Component {
         } = this.props;
 
         return (
-            <Card className='ins-c-card__compliance'
+            <TemplateCard className='ins-c-card__compliance'
                 { ...complianceFetchStatus !== 'pending' ? {
                     'data-ouia-safe': true
                 } : { 'data-ouia-safe': false } }
             >
-                <CardHeader>
-                    <Title size={ 'lg' }>Compliance</Title>
-                </CardHeader>
-                <CardBody>
+                <TemplateCardHead>
+                    <TemplateCardActions downloadReport="true">
+                    </TemplateCardActions>
```

Comment:
this can be made into a one sided tag:

`<TemplateCardActions downloadReport="true"/>`

I'm not going to comment this on every one, but let's convert them to this if there are no children.